### ISSUE TITLE
zeropadding problem

### ISSUE THREAD

**Issue by siemonchan** [qqwweee/keras-yolo3#274]
I`m confused by author`s using zeropadding layer and 'valid' padding convolution instead of 'same' padding in 'model.py'.
In 'convert.py' author said that this is because darknet uses left and top padding, but when using 'same' padding in tensorflow, I personally can not find any differences in output. Maybe there is some other reason that I haven`t noticed.
